PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py

```diff
@@ -59,20 +74,7 @@ def applyDeepBtagging( process, postfix="" ) :
        elSource = cms.InputTag('slimmedElectrons'),
        rParam = 0.8,
        jetCorrections = ('AK8PFPuppi', cms.vstring(['L2Relative', 'L3Absolute']), 'None'),
-       btagDiscriminators = [
-          'pfDeepDoubleBvLJetTags:probQCD',
-          'pfDeepDoubleBvLJetTags:probHbb',
-          'pfDeepDoubleCvLJetTags:probQCD',
-          'pfDeepDoubleCvLJetTags:probHcc',
-          'pfDeepDoubleCvBJetTags:probHbb',
-          'pfDeepDoubleCvBJetTags:probHcc',
-          'pfMassIndependentDeepDoubleBvLJetTags:probQCD',
-          'pfMassIndependentDeepDoubleBvLJetTags:probHbb',
-          'pfMassIndependentDeepDoubleCvLJetTags:probQCD',
-          'pfMassIndependentDeepDoubleCvLJetTags:probHcc',
-          'pfMassIndependentDeepDoubleCvBJetTags:probHbb',
-          'pfMassIndependentDeepDoubleCvBJetTags:probHcc',
-       ] + pfDeepBoostedJetTagsAll,
+       btagDiscriminators = _btagDiscriminators.names,
```

Comment:
I'd suggest
```suggestion
       btagDiscriminators = _btagDiscriminators.names.value(),
```
Then, if anyone would modify `_btagDiscriminators.names` later in this function, those changes would **not** propagate here (as would happen now). Unless that would be the intention, of course.

And as an added value, then the right-hand-side would be without the `cms.` type so the "typo checking" would still be done.